recipes/xdmfwrite_highfive/bld.bat

```diff
@@ -0,0 +1,13 @@
+
+cmake -G"NMake Makefiles" ^
+  -DCMAKE_INSTALL_PREFIX:PATH="%LIBRARY_PREFIX%" ^
+  -DCMAKE_PREFIX_PATH:PATH="%LIBRARY_PREFIX%" ^
+  -DCMAKE_BUILD_TYPE:STRING=Release ^
+  .
+if errorlevel 1 exit 1
```

Comment:
Isn't it typical to make a build dir for cmake?

Reply:
Indeed. But since it is only an install build and the close is directly deleted it is not very important. In changed it to a build dir anyway